Presentation.Web/app/components/it-contract/tabs/it-contract-tab-main.controller.ts

```diff
@@ -307,6 +328,54 @@
                     }
                 }
 
+                function bindCriticalities() {
+                    
+                    const optionMap = criticalityOptions.reduce((acc, next, _) => {
+                        acc[next.Id] = {
+                            text: next.Name,
+                            id: next.Id,
+                            optionalObjectContext: {
+                                id: next.Id,
+                                name: next.Name,
+                                description: next.Description
+                            }
+                        };
+                        return acc;
+                    }, {});
+
+                    //If selected state is expired, add it for presentation reasons
+                    let existingChoice = null;
+                    if ($scope.contract.criticalityTypeId !== undefined && $scope.contract.criticalityTypeId !== null) {
+                        existingChoice = {
+                            id: $scope.contract.criticalityTypeId,
+                            name: $scope.contract.criticalityTypeName
+                        };
+
+                        if (!optionMap[$scope.contract.criticalityTypeId]) {
+                            optionMap[existingChoice.id] = {
+                                text: existingChoice.name,
+                                id: existingChoice.id,
+                                disabled: true,
+                                optionalObjectContext: existingChoice
+                            }
+                        }
+                    }
+
+                    const options = criticalityOptions.map(option => optionMap[option.Id]);
+
+                    $scope.criticality = {
+                        selectedElement: existingChoice && optionMap[existingChoice.id],
+                        select2Config: select2LoadingService.select2LocalDataNoSearch(() => options, true),
+                        elementSelected: (newElement) => {
+                            if (!newElement)
```

Comment:
this is why clearing the selection does not work

Reply:
See how it's done in DPR 
![image](https://user-images.githubusercontent.com/13116889/175811149-9dd6a784-57fa-4b71-ba35-b88339bfbd88.png)